synapse/lib/node.py

```diff
@@ -732,12 +732,13 @@ def tags(pode, leaf=False):
 
     Args:
         pode (tuple): A packed node.
-        leaf (bool): If True, only return the full tags.
+        leaf (bool): If True, only return leaf tags and tags that have values
```

Comment:
This could cause issues if you really do want only the leaf tags.  It feels like we might be ditching one use case to facilitate another.

Reply:
there is a not-insubstantial amount of code using the packed node helpers. I'd suggest making a new helper function if needd; rather than changing an existing one.